lib/sequences/06_argonaut_profiles_sequence.rb

```diff
@@ -30,6 +29,24 @@ class ArgonautProfilesSequence < SequenceBase
     assert errors.empty?, "Patient did not validate against profile: #{errors.join(", ")}"
   end
 
+  test 'Patient has address',
+          'Additional Patient resource requirement' do
+
+    assert !@patient.nil?, 'Expected valid DSTU2 Patient resource to be present'
+    assert @patient.is_a?(FHIR::DSTU2::Patient), 'Expected resource to be valid DSTU2 Patient'
+    telecom = @patient.try(:address).try(:first)
+    assert telecom, 'Patient address not returned'
```

Comment:
Misleading variable name.  Copy paste error?  

Also, it's a good idea to be explicit and say 

```assert !telecom.nil?, 'Patient address not returned.'```

... in case for some reason this variable can evaluate to false in a valid situation.